src/app/globals.css

```diff
@@ -166,7 +166,7 @@
 /* Custom Scrollbar Styling */
 /* For Webkit Browsers */
 ::-webkit-scrollbar {
-  width: 14px; /* Width of the scrollbar */
```

Comment:
Is this change necessary????????????????????????????

I would overlook though

Reply:
yh, the scrollbar is taking too much real estate on the mobile view, reducing the bar width by 2px solves the issue.